setup.py

```diff
@@ -45,7 +45,7 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket >= 1.33.0",
+        "pytket >= 1.34.0",
```

Comment:
Do you want to put this in the changelog?
Maybe adding support for the new clexp as well?